flink-table/flink-sql-jdbc-driver/src/main/java/org/apache/flink/table/jdbc/utils/DatabaseMetaDataUtils.java

```diff
@@ -66,7 +67,8 @@ public static FlinkResultSet createCatalogsResultSet(
         return new FlinkResultSet(
                 statement,
                 new CollectionResultIterator(catalogs.iterator()),
-                ResolvedSchema.of(TABLE_CAT_COLUMN));
+                ResolvedSchema.of(TABLE_CAT_COLUMN),
+                ZoneId.systemDefault());
```

Comment:
For `createCatalogsResultSet` and `createSchemasResultSet`, if it's no need to set the zone id (I guess there is no `timestamp` type for these two cases, right?), it's better to add a comment to say it.